packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -870,13 +873,13 @@ class VideoViewport extends Viewport implements IVideoViewport {
       canvasPos[1] - panOffsetCanvas[1],
     ];
 
-    const worldPos: Point3 = [
-      subCanvasPos[0] / worldToCanvasRatio,
-      subCanvasPos[1] / worldToCanvasRatio,
+    destPos.splice(
```

Comment:
i'm not sure what is happening here

Reply:
The first two values are getting replaced to agree with the contract coming from vtk - which we use in 1 place.